weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooks.ts

```diff
@@ -569,6 +569,26 @@ const useCallsStats = (
   }, [callStatsRes, opts?.skip, refetch]);
 };
 
+/*
+  Helper that calls the call stats hook with limit: 1, returning a boolean
+  if there are any calls in the project. This uses a highly optimized
+  query in the backend. 
+*/
+const useProjectHasCalls = (
+  entity: string,
+  project: string,
+  opts?: {skip?: boolean}
+): Loadable<boolean> => {
+  const callsStats = useCallsStats(entity, project, {}, undefined, 1, opts);
+  return useMemo(() => {
+    return {
+      loading: callsStats.loading,
+      result: callsStats.result != null,
```

Comment:
I notice that this API allows limit 0 which seems weird.

Reply:
```python
if limit < 0:
            raise ValueError("Limit must be a positive integer")
```

huh... yeah. `0` is not a positive integer either.... weird

Reply:
ill fix that